src/logics/Logic.cc

```diff
@@ -532,7 +532,7 @@ PTRef Logic::mkXor(vec<PTRef>&& args) {
     else if (args[0] == getTerm_false() || args[1] == getTerm_false())
         return (args[0] == getTerm_false() ? args[1] : args[0]);
 
-    sort(args);
```

Comment:
Do we need to sort the arguments?
Shouldn't this be handled in `mkFun`?

Reply:
Right now it isn't handled there.  Whether it should be done there is another question.  Quickly thinking not doing this in `mkFun` could allow additional optimisations when we are merging two already sorted boolean operators.